keepassxc-browser/content/fill.js

```diff
@@ -166,6 +173,7 @@ kpxcFill.fillTOTPFromUuid = async function(el, uuid) {
 // Set normal or segmented TOTP value
 kpxcFill.setTOTPValue = function(elem, val) {
     if (kpxc.combinations.length === 0) {
+        logDebug('Error: Credential list is empty.');
```

Comment:
Given that the code+line is not printed, it would be very helpful to add context to these messages. (Is it possible to add file+line to the debug messages??)

Context for this one would be: Totp Fill :: No credentials found :: (optional why if known)

Reply:
The problem with these messages are that now when the log function is always the same, JavaScript Console doesn't show the line where the error is coming from.. true. The information must be added manually to these messages. There are some automatic options to retrieve the info, and I'm gonna try them first.

Reply:
Yah I don't think we need/want a stack trace and we don't want to manually hardcode a location. Worst case we just add a little context to give a hint as to where we are in the sequencing.

Reply:
I added a new commit that also reports the file and line number. Just a simple parsing of the current stack. Seems there's no other way to do that, but I think that's good enough.